src/hooks/useDebounceString.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from 'react';
+
+export default function useDebounceString(value: string, delay: number = 200) {
```

Comment:
We already have the dep `use-debounce` which does something very similar. Can we use that instead of creating this new hook?

Reply:
didn't realise! i'll convert!